[excel] (Comments) Adding snippets for comment APIs

AlexJerabek opened the pull request.

kbrandl commented:
@AlexJerabek sorry for the slow response on this one -- would it be possible for you to have someone else review it? I'm OOF for the remainder of the week, starting in another half hour, and rushing to wrap up other things before then. I'd be happy to review it on Monday when I'm back in the office, if you're still awaiting review then. Thanks!

Rick-Kirkham commented on samples/excel/85-preview-apis/comment.yaml:
```diff
@@ -0,0 +1,167 @@
+        async function addCommentReply() {
+            await Excel.run(async (context) => {
+                const comment = context.workbook.comments.getItemByCell("Comments!A1");
+                context.load(comment);
```
I'd like to see a code comment explaining why the comment object has to be loaded before you can add to its replies collection. Also, `context.load(comment)` is equivalent to `comment.load()', so its an empty load that causes all of the comment's scalar properties to be loaded. Is that really necessary?

Rick-Kirkham commented on samples/excel/85-preview-apis/comment.yaml:
```diff
@@ -0,0 +1,167 @@
+    ExcelApi: 1.9
+script:
+    content: |
+        $("#setup").click(() => tryCatch(setup)); $("#addCommentToSelectedCell").click(() => tryCatch(addCommentToSelectedCell)); $("#addCommentToCell").click(() => tryCatch(addCommentToCell)); $("#addCommentReply").click(() => tryCatch(addCommentReply)); $("#getCommentMetadata").click(() => tryCatch(getCommentMetadata)); $("#editComment").click(() => tryCatch(editComment)); $("#editCommentReply" ...
```
Nit: Doesn't seem to be a line break at the semi-colons in the button handler assignments.

AlexJerabek commented:
Thanks @Rick-Kirkham. The first load statement in addCommentReply isn't actually necessary. Good catch. I've removed it and updated the PR accordingly.
